.github/workflows/build_and_publish.yaml

```diff
@@ -66,7 +66,9 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v1
     - name: Build image
-      run: docker build . --file Dockerfile --tag $IMAGE_NAME  --build-arg=pythonversion=3.8.3
+      run: make docker-build
+    - name: Test image
+      run: make docker-test ARGS='-e PRIVATE_REPO_ACCESS_TOKEN'
```

Comment:
Because we use `env_file: ../.env` in the docker-compose.yaml, there is an invalid PRIVATE_REPO_ACCESS_TOKEN set, which causes a couple of tests to fail.

It maybe better to add a `.test.env` file that is used by docker-compose, perhaps, but we would need to generate that and include the PRIVATE_REPO_ACCESS_TOKEN and possibly others.